nni/retiarii/oneshot/pytorch/proxyless.py

```diff
@@ -209,7 +222,79 @@ def _train_one_epoch(self, epoch):
                 _logger.info("Epoch [%s/%s] Step [%s/%s]  %s", epoch + 1,
                              self.num_epochs, step + 1, len(self.train_loader), meters)
 
-    def _logits_and_loss(self, X, y):
+    def _form_latency_table(self, dummy_input):
```

Comment:
Can we put this into `LatencyPredictor`?

Reply:
Do you mean define a class named `LatencyPredictor` and refer it to form latency table and get expected latency? Should it be a subclass of `ProxylessTrainer` or a separate class?

Reply:
I think you already have a class named LatencyPredictor?

I'm not sure. But putting latency-related logic in proxylessnas trainer looks very strange. If you have another trainer (let's say SPOS trainer or FBNet trainer or DARTS trainer or ENAS trainer) who has a similar need for latency calculation, do you need to write all these logic over again?